src/misc/falling-blocks.ts

```diff
@@ -4,6 +4,11 @@ import { Leaves } from 'org.bukkit.block.data.type';
 import { FallingBlock } from 'org.bukkit.entity';
 import { EntityChangeBlockEvent } from 'org.bukkit.event.entity';
 
+/**
+ * Falling blocks that don't break any blocks.
+ */
+const LIGHT_BLOCKS = [Material.SNOW];
+
```

Comment:
I think we should use ES6 Set instead of an array of Materials, but it doesn't matter here.